examples/data-objects/table-document/package.json

```diff
@@ -99,8 +99,6 @@
 		}
 	},
 	"typeValidation": {
-		"disabled": true,
-		"version": "2.0.0-internal.2.1.0",
-		"broken": {}
+		"disabled": true
```

Comment:
Is this correct to be disabled?  In theory we have some compat burden here (though not sure it's really worth worrying about).

Reply:
It's in the @fluid-example scope; is there some reason this example is different from others?

Reply:
It's imported by Loop (historical backcompat reasons)